awx/main/tasks/jobs.py

```diff
@@ -107,12 +105,15 @@ class BaseTask(object):
     abstract = True
```

Comment:
Right here I would prefer to do

```python
    self.callback_class = RunnerCallback
```

Then in RunProjectUpdate, put in the respective callback class

Reply:
I have made the required change.